### DISCUSSION TITLE
More than 1 scripting tab open at the same time

### DISCUSSION THREAD

**Discussion by microlemon** [godotengine/godot-proposals#10531] [General Discussions]
please make it so you can open at least another tab to see code in more than one script at the same time. I don't want to have to use vscode because string literals aren't recognized and auto suggested as well as godot's ide. also would be nice to be able to resize the script windows font sizes not just the script font with ctrl+=
I don't know if this is the right place the post this :p

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/414

**Comment by ReptarJerky** [comment]
@microlemon hello, I'm not sure if you're the same one but did you create the old game, Chase The Dot? Sorry if this is unrelated to this thread and such..

**Reply by microlemon** [reply]
nope, sorry
